sigmastate/src/main/scala/sigmastate/interpreter/ProverResult.scala

```diff
@@ -13,6 +13,7 @@ import sigmastate.utils.{SigmaByteReader, SigmaByteWriter}
   * @param extension - user-defined variables to be put into context
   */
 class ProverResult(val proof: Array[Byte], val extension: ContextExtension) {
+  // Usage of Ints.fromByteArray isn't allowed, as `proof` can be empty array
```

Comment:
not only that, but the proof prefix bytes are not random, so applying this optimization doesn't make sense. Please remove the comment.

Reply:
Removed in new version. The comment was put there to explain why we use Ints.fromByteArray everywhere in the codebase except these 2 usages.